<ui version="4.0" >
 <class>TextAnalysisDialog</class>
 <widget class="QDialog" name="TextAnalysisDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Text Analysis</string>
  </property>
  <layout class="QGridLayout" >
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>actions</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QPushButton" name="countButton" >
        <property name="text" >
         <string>count</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton" >
        <property name="text" >
         <string>close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>info</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>total lines</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="totalLinesLabel" >
        <property name="text" >
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>text lines</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>total shown phrases</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>comment lines</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="textLinesLabel" >
        <property name="text" >
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="commentLinesLabel" >
        <property name="text" >
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLabel" name="displayedWordLabel" >
        <property name="text" >
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>different phrases</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QLabel" name="differentWordLabel" >
        <property name="text" >
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>characters in phrases</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QLabel" name="characterInWordsLabel" >
        <property name="text" >
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="3" row="0" column="1" colspan="3" >
    <widget class="QTableView" name="resultView" />
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>options</string>
     </property>
     <layout class="QFormLayout" name="formLayout" >
      <property name="fieldGrowthPolicy" >
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="normalTextCheck" >
        <property name="text" >
         <string>normal text</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="commentsCheck" >
        <property name="text" >
         <string>comments</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="commandsCheck" >
        <property name="text" >
         <string>commands</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QComboBox" name="minimumLengthMeaning" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex" >
         <number>1</number>
        </property>
        <item>
         <property name="text" >
          <string>show all phrases</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>minimum phrase length:</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>minimum word length (one):</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>minimum word length (all):</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>fully ignore shorter words:</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QSpinBox" name="minimumLengthSpin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="value" >
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>words per phrase:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QSpinBox" name="sentenceLengthSpin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>minimum count:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QSpinBox" name="minimumCountSpin" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>source:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" >
       <widget class="QComboBox" name="comboBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text" >
          <string>whole document</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>selection</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
